.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+- "node"
+- "4.2"
+before_install:
+- rvm install 2.2.3
```

Comment:
Actually, calling `rvm` directly like this doesn't work with `stable`. But I've updated it to use 2.3.0 explicitly.